Feature/#29

schatzen opened the pull request:
### 작업 내용
1. PhoneAuthField 컴포넌트 재사용
2. 개인정보 수정 Form 모델 생성
3. 휴대폰 인증 완료 시 수정버튼 기능 활성화

schatzen committed:
feat: TextField 컴포넌트 구현 (#29)

schatzen committed:
feat: 휴대폰 본인 인증 컴포넌트 구현 (#29)

schatzen committed:
feat: 타이틀 및 상세설명 컴포넌트 구현 (#29)

schatzen committed:
fix: 페이지 세팅정보 업로드 (#29)

schatzen committed:
feat: TextFieldWithTItle props 추가 및 부모에 textfield State 추가

schatzen committed:
fix: 상세설명 opacity 수정 (#29)

schatzen committed:
feat: PhoneAuthField 컴포넌트 재사용 (#29)

schatzen committed:
fix: FieldHeader 컴포넌트 디자인 및 props 수정

schatzen committed:
feat: 개인정보수정 root단 전역 state 추가 및 수정버튼 validation 추가 (#29)

schatzen committed:
fix: 이메일 validation시 수정버튼 활성화되는 기능 제거 (#29)

schatzen committed:
fix: Form 모델 외부로 이전 (#29)

schatzen committed:
fix: stories 제거 및  수정 (#29)

schatzen committed:
fix: 이메일 validation 통과 못할시 수정버튼 비활성화 (#29)

Commit:
fix: 휴대폰 인증 컴포넌트 중복 제거, my-page 라우팅 경로 수정 (#29)

owl094 commented on src/domains/MyPage/Edit/components/EditView/EditView.tsx:
```diff
@@ -0,0 +1,159 @@
+import { FieldHeader } from "../FieldHeader";
+import { PhoneAuthFields } from "@/domains/Signup/BasicInfo/components/PhoneAuthFields";
+
+export type NameAlertProps = {
```
❌ 사용하지 않는 코드 블럭 제거 바랍니다.

owl094 reviewed (changes requested):
`EditView.tsx - 15~17 Line` 제거 필요
Reason: 사용하지 않는 코드

Commit:
fix: unused type (#29)

미사용 코드 제거

ref:
https://github.com/Dream-Maker-Inc/ureca-frontend/pull/40#discussion_r859722969

Commit:
chore: node version up
